server/src/routes/api/v1/messageRouter.js

```diff
@@ -0,0 +1,14 @@
+import express from "express"
+import { Message } from "../../../models/index.js"
+
+const messageRouter = new express.Router()
+
+messageRouter.get("/", async (req,res) => {
+    try {
+        const messages = await Message.query()
+        return res.status(200).json({messages:messages})
```

Comment:
we should space pad inside an object `{ messages:messages }` same below for errors